docs/CONTRIBUTING.md

```diff
@@ -24,6 +24,105 @@ sdkmanager \
     "tools"
 ```
 
+# Adding new libraries
```

Comment:
Wow, great job describing it! I have good candidates for "playgrounding" with this guide - Fresco and Lottie.